tests/main/ubuntu-core-snap-persists/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: Ensure a snap is still there after reboot
```

Comment:
That's a bit of a strange test. Feels too expensive for what it is testing.

Can we merge this check into something else that needs to reboot for better reasons?  The service test above, maybe?